app/assets/stylesheets/common/_markus.scss

```diff
@@ -1366,3 +1363,54 @@ canvas {
   min-width: 250px;
   width: 50%;
 }
+
+.exam-crop-container {
+  display: flex;
+  height: 600px;
+  width: 1000px;
+}
+
+#exam-crop-img-container {
+  background-color: #f1f1f1;
```

Comment:
You shouldn't introduce new colour constants here, but instead reuse the existing ones. If you're worried about the colour being too light you can put a border or shadow around the container

Reply:
I'm actually worried about the colour of `$disabled_area` being too dark (since `Jcrop` greys out the image when there's a crop selection saved), and I didn't want to use a constant that's unrelated here (like `$line`). Do you have any suggestions?

Reply:
Can you share a screenshot to illustrate the look you're worried about?

Reply:
Looking at the images side by side, I'm realizing that it's not as big of a distinction between my chosen colour and the one specified by `$disabled_area`. 

### Background with custom colour
<img width="976" alt="Background with custom colour" src="https://github.com/MarkUsProject/Markus/assets/79552425/8808f051-9666-4063-a50d-7c79fb547dce">

### Background with $disabled_area colour
<img width="986" alt="Background with $disabled_area colour" src="https://github.com/MarkUsProject/Markus/assets/79552425/7eb27c9a-a628-4128-81dc-229cbb7ce914">

Reply:
I've pushed changes to use `$disabled_area` instead. I'm going to wait for your confirmation that the colour is not too dark before I resolve this conversation, however.

Reply:
@anissingh okay thanks for uploading those screenshots. Yeah, I think `$disabled_area` isn't too dark, so I'm good with sticking with that 😄